addons/web_editor/static/src/js/widgets/ace.js

```diff
@@ -562,12 +564,18 @@ var ViewEditor = Widget.extend({
         }).bind(this));
         if (errorFound) return $.Deferred().reject(errorFound);
 
-        var defs = [];
+        var mutex = new concurrency.Mutex();
         _.each(toSave, (function (_toSave, type) {
-            defs = defs.concat(_.map(_toSave, (type === 'xml' ? this._saveView : this._saveSCSS).bind(this)));
+            // Child views first as COW on parent will delete it
```

Comment:
"Child views first as COW on **a** parent **would** delete **them**" :wink:

Reply:
cow should not delete, but move view :'(

Reply:
Don't worry, it should be done in saas11.3 for the release of stable multi website